GOVERNANCE.md

```diff
@@ -72,8 +72,10 @@ membership beyond these rules.
 
 The CTC may add additional members to the CTC by a standard CTC motion.
 
-A CTC member may be removed from the CTC by voluntary resignation, or by
-a standard CTC motion.
+When a CTC member's participation in CTC matters has become minimal for a
```

Comment:
Since it was part of the question, I guess i should also put out my opinion of what constitutes CTC matters (responsibilities of CTC members) as distinct from working on the Node.js project (responsibilities of all Collaborators).

To participate meaningfully in the CTC, one might:

* routinely attend the weekly meeting
* regularly provide valuable comments on the weekly CTC meeting issue (the one that Rod opens that contains the agenda ahead of the meeting) and/or on issues labeled `ctc-agenda`
* participate meaningfully on CTC email threads
* take on action items (that is, volunteer for tasks) that come out of CTC discussions (meeting discussions, email threads, etc.)

That list is probably not exhaustive. It is also subject to some hair-splitting. (Is it enough to "attend" the weekly meeting, or should one be expected to "participate" in the meeting?) Nonetheless, it is hopefully indicative of the types of things I have in mind, and I hope it generally conforms to what others have in mind, even if there are some specifics that may vary.

In contrast, the follow work, while absolutely essential to the project and of extremely high value, is not particular to the CTC and is in fact work that is ideally done by all Collaborators:

* comments on issues
* comments on PRs
* helping other users and/or novice contributors
* contributing code and/or documentation changes to improve the project
* participation in working groups
* landing PRs

Again, doing all that stuff is hugely valuable. However, it's not CTC work. Not all acts of awesomeness constitute CTC work. (If things are going well, chances are CTC work will be distinctly unglamorous, in fact.)